docker/.gitpod.Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM gitpod/workspace-full
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -qq \
```

Comment:
Adding  `sudo` to `apt-get update`, `apt-get install`, and `rm` gets past the initial error.

Reply:
We can just bump the USER and drop it at the end